### PR TITLE
Update

### PR DESCRIPTION
**Pull request by souptik-samanta**
Updating for-reviewers.hackclub.com

I dont know the website was showing not found so chaged the value

### CODE REVIEW

**Review by alx-alexpark — CHANGES_REQUESTED** [review]
Hi, could you sort your record alphabetically within the file?

**Comment by souptik-samanta** [comment]
@alx-alexpark did it

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2459,6 +2459,10 @@ som-mail-system:
   - ttl: 600
     type: CNAME
     value: cname.vercel-dns.com.
+souptik:
+   - ttl: 600
+     type: CNAME
+     value: souptik-samanta.github.io/.
```
Comment (Muirrum) [inline hackclub.com.yaml:2465]:
```suggestion
     value: souptik-samanta.github.io.
```

DNS records can't have paths (`/`)

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (souptik-samanta) [reply hackclub.com.yaml:2465]: I will do it and let you know
One issue could you go to the link I have in value that link is working when going from browser but 
Previously you had approved and merger it but when I was going to the 
Subdomain.hackclub.com(The subdomain is what I have that time) it was showing not found but was redirecting to gihub pages where a 404 error code was showing what might be the issue ?
Please let me know and I will remove the /
@/muirrum

Comment (Muirrum) [reply hackclub.com.yaml:2465]: you need to configure the CNAME in github pages.

**Comment by souptik-samanta** [comment]
Hey i commited the as you suggested also gave hackclub.com in git pages customain doamin 
Its ok yes?

**Comment by souptik-samanta** [comment]
@Muirrum did it as you said in custom domain gave hackclub.com this error 
after giving souptik.hackclub.com this error
image
same that yellow box was showing should that be there?

**Comment by Muirrum** [comment]
That's because the record doesn't exist yet. Why is this souptik.hackclub.com instead of for-reviewers.hackclub.com?

**Comment by souptik-samanta** [comment]
I changed that to give my name idk I just liked it but it's still for the reviewers only I will use this to host the page that we guys made for the reviewers 
I don't know I didn't like the thing "for reviewers "one 
So I just gave it

**Comment by souptik-samanta** [comment]
When would it be merged?

**Comment by Muirrum** [comment]
Stick to the for-reviewers name. This feels a little too close to a personal site for hackclub.com.

**Comment by souptik-samanta** [comment]
Okay as you say 
So should I close the pr 
And will I get the for-reviewers.hackclub.com?
And just put it in that customs domain ?

**Comment by souptik-samanta** [comment]
Okay as you say 
So should I close the pr 
And will I get the for-reviewers.hackclub.com?
And just put it in that customs domain ?
 I will close the pr tomorrow morning I am currently not at my pc

**Comment by Muirrum** [comment]
Yes. you should use for-reviewers.hackclub.com.

**Comment by souptik-samanta** [comment]
Thank you for closing!

**Pull request closed without merging**